src/main/java/com/thullo/service/TaskServiceImpl.java

```diff
@@ -71,14 +71,14 @@ private String uploadTaskFile(MultipartFile file, String requestUrl) throws BadR
 
     @Override
     public Task moveTask(String boardRef, String status, Long position) throws ResourceNotFoundException {
-        Status statusToUpdateTo = Status.getStatus(status);
+
         Task task = taskRepository.findByBoardRef(boardRef)
                 .orElseThrow(() -> new ResourceNotFoundException("Task not found"));
 
-        List<Task> tasksInStatus = taskRepository.findAllByBoardAndStatus(task.getBoard(), statusToUpdateTo);
+        List<Task> tasksInStatus = taskRepository.findAllByBoardAndStatus(task.getBoard(), status);
```

Comment:
Don't trust what is coming from the request, format it before you call the repository method

Reply:
Resolved...